tower/Cargo.toml

```
@@ -48,9 +48,9 @@ hedge = ["util", "filter", "futures-util", "hdrhistogram", "tokio/time", "tracin
 limit = ["tokio/time", "tokio/sync", "tokio-util", "tracing", "pin-project-lite"]
 load = ["tokio/time", "tracing", "pin-project-lite"]
 load-shed = ["pin-project-lite"]
-make = ["pin-project-lite", "tokio/io-std"]
+make = ["pin-project-lite", "tokio/sync"]
```

I don't see any use of `tokio::sync` in `make/`, so I think we don't need this feature activation either.

```suggestion
make = ["pin-project-lite", "tokio"]
```

---

We seem to have `sync` as an enabled feature of `tokio` whenever that optional dependency is activated, further below. We should be able to drop that as well.

---

Ah, indeed. Fixed.